File: Csharp/Lab8/Lab8/Lab8/cng/Model/Collections/neww/CngCollection.cs
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace Lab8.cng.Model.Collections.neww
{
    public interface ICngCollection<T> : IEnumerable<T>
    {
        int Size();
        bool IsEmpty();
        void Add(T t);
        void AddAll(Collection<T> c);
        bool Contains(T t);
        bool ContainsAll(Collection<T> c);
        object[] ToArray();
    }
}

File: Java/Desktop/Saod/Lab3/src/by/bntu/fitr/prostrmk/collections/CngCollection.java
package by.bntu.fitr.prostrmk.collections;

import java.util.Collection;

public interface CngCollection<T> extends Iterable<T>{

    int size();
    boolean isEmpty();
    void addAll(Collection c);
    boolean contains(Object o);
    Object []toArray();
    boolean containsAll(Collection c);
    void clear();



}
